stock_return_request/models/stock_return_request.py

```diff
@@ -334,8 +331,9 @@ def _action_confirm(self):
             raise ValidationError(
                 _(
                     "It wasn't possible to assign stock for this returns:\n"
-                    "%s" % failed_moves_str
+                    "%(failed_moves_str)s"
                 )
+                % {"failed_moves_str": failed_moves_str}
```

Comment:
Usage of format() is better

Reply:
Even for translatable content?

Reply:
Of course, you can do:

_("My string {variable1}, {variable2}").format(variable1="Test", variable2="Test2")

Reply:
Is that preference expressed in any place of the guidelines?

Reply:
It's not an OCA one, but a python one

https://realpython.com/python-f-strings/

Reply:
But we can do a PR on OCA contributing guidelines

Reply:
f-strings is something different than `.format`. What are the advantages of using `format` over `%`?

Reply:
For me, readability

Reply:
Using dictionaries makes the readability fairly the same, and with `%`, you have less chances of overpassing line length, so not sure it's an advantage.